scripts/training/prepare-config.py

```diff
@@ -86,3 +86,90 @@
     yaml.dump(config, yaml_file, default_flow_style=False, default_style='\'', explicit_start=True)
 
 s3_client.upload_file(Bucket=s3_bucket, Key=yaml_key, Filename=local_yaml_path)
```

Comment:
Last comment - I think: The file without the number should only be uploaded if we are NOT having multi-config.

Reply:
yes ok.  once all is working properly, easiest probably just to move this down to an else statement after multi_config if statement.   need it here now to troubleshoot.

Reply:
this has now been completed